custom_components/tuya_local/devices/pinguino_portable_airconditioner.yaml

```diff
@@ -0,0 +1,125 @@
+name: Pinguino
+products:
+  - id: anR7aeh1YPTKAz5a
+    name: Pinguino PACEL140LRFK 3ALBK
+primary_entity:
+  entity: climate
+  dps:
+    - id: 1
+      name: hvac_mode
+      type: boolean
+      mapping:
+        - value: "off"
+        - dps_val: false
+          value: "off"
+        - dps_val: true
+          constraint: preset_mode
+          conditions:
+            - dps_val: null
+              value: "off"
+            - dps_val: "1"
+              value: dry
+            - dps_val: "2"
+              value: cool
+            - dps_val: "3"
+              value: fan_only
+            - dps_val: "4"
+              value: auto
+    - id: 5
+      name: preset_mode
+      type: string
+      mapping:
+        - dps_val: "4"
+          value: eco
+    - id: 6
+      type: integer
+      name: temperature
+      range:
+        min: 18
+        max: 32
+      mapping:
+        - constraint: temperature_unit
+          conditions:
+            - dps_val: true
+              range:
+                min: 64
+                max: 90
+    - id: 8
+      type: string
+      name: fan_mode
+      mapping:
+        - value: auto
+        - dps_val: "0"
+          value: auto
+        - dps_val: "1"
+          value: auto
+          constraint: preset_mode
+          conditions:
+            - dps_val: "2"
+              value: low
+            - dps_val: "3"
+              value: low
+        - dps_val: "2"
+          value: auto
+          constraint: preset_mode
+          conditions:
+            - dps_val: "2"
+              value: medium
+            - dps_val: "3"
+              value: medium
+        - dps_val: "3"
+          value: auto
+          constraint: preset_mode
+          conditions:
+            - dps_val: "2"
+              value: high
+            - dps_val: "3"
+              value: high
+    - id: 16
+      name: swing_mode
+      type: boolean
+      mapping:
+        - value: "off"
+        - dps_val: false
+          value: "off"
+        - dps_val: true
+          value: vertical
+          constraint: hvac_mode
+          conditions:
+            - dps_val: false
+              value: "off"
```

Comment:
Is this really necessary? It is normal for settings to be retained when a device is off. I don't think it is necessary to show swing_mode as off just because the device is off.

Reply:
Removed.